### ISSUE TITLE
🐛 [firebase_ml_vision] Barcode engine is nil.

### ISSUE THREAD

**Issue by irizzka** [FirebaseExtended/flutterfire#6058]
Flutter 2.0.5 • channel stable 
Dart 2.12.3

firebase_core: 1.1.0
firebase_ml_vision: 0.12.0+1   

Barcode engine is nil.

if I add  pod 'Firebase/MLVisionBarcodeModel' to  # ...
target 'Runner' do

I stuck with this error

[!] CocoaPods could not find compatible versions for pod "Firebase/CoreOnly":
  In Podfile:
    Firebase/MLVisionBarcodeModel was resolved to 5.0.0, which depends on
      Firebase/CoreOnly (= 5.0.0)

    firebase_core (from `.symlinks/plugins/firebase_core/ios`) was resolved to 1.1.0, which depends on
      Firebase/CoreOnly (= 7.11.0)

Barcode scanning |
 Firebase/MLVisionFirebase/MLVisionBarcodeModel     - DEPRECATED 
 GoogleMLKit/BarcodeScanning

**Comment by markusaksli-nc** [comment]
Hi @irizzka 
This issue is caused by not having the barcode model in podspec and will be blocked by https://github.com/FirebaseExtended/flutterfire/issues/4625 on iOS
Closing since this is pretty much a duplicate of https://github.com/FirebaseExtended/flutterfire/issues/4625
Thank you

**Comment by irizzka** [comment]
Fork is not an option
